### ISSUE TITLE
Upgrading from prawn-styled-text

### ISSUE THREAD

**Issue by brandoncc** [blocknotes/prawn-html#49]
Hi, thanks for these awesome libraries!

I would like to upgrade from `prawn-styled-text`, and I'm wondering if you can give one or both of the following:

- A comprehensive list of what elements render differently in `prawn-html` vs `prawn-styled-text`.
- A stylesheet that will reset `prawn-html` to render the same as `prawn-styled-text` by default.

We have an application with a lot of content that gets rendered with `prawn-styled-text` right now and I'm interested in switching. It is an absolute requirement to have rendering parity though. Is this possible?

Thanks again.

**Comment by blocknotes** [comment]
Hey @brandoncc.
I decided to create `prawn-html` from scratch because of some design limitations that I found in `prawn-styled-text` and refactoring the old project would have been a massive effort.

`prawn-html` internally works in a completely different way from the previous project because it handles each element styles with its own set of CSS rules. `prawn-styled-text` is more "custom" on this side and it lacks features like margin collapsing or CSS styles.

For these reasons, unfortunately, I don't see an easy way to upgrade contents used with the old project.
